dubbo-config/dubbo-config-spring/src/main/java/org/apache/dubbo/config/spring/beans/factory/annotation/ServiceClassPostProcessor.java

```diff
@@ -78,7 +78,7 @@
 import static org.springframework.util.ClassUtils.resolveClassName;
 
 /**
- * {@link BeanFactoryPostProcessor} used for processing of {@link Service @Service} annotated classes. it's also the
+ * {@link BeanFactoryPostProcessor} used for processing of {@link DubboService @Service} annotated classes. it's also the
```

Comment:
Does @Service have to be changed to @DubboService?

Reply:
I think it's necessary if you add @Deprecated on @Service.
Obviously, there is no business value and increase commiters' burden. I'll colse it if nobody agree with me.